adam/experiment/observer.py

```diff
@@ -311,6 +312,9 @@ def post_observer_log(
                 f"\t\t\t\t<td>\n"
                 f'\t\t\t\t\t<h3 id="perception-{instance_number}">Learner Perception</h3>\n'
                 f"\t\t\t\t</td>\n"
+                f"\t\t\t\t\t<td>\n"
```

Comment:
We can to be able to turn on/off whether the "scene renderings" column is included via a parameter.  (This should probably be stored as a field on `DescriptionObserver`).